moduleframework/common.py

```diff
@@ -674,7 +612,7 @@ def run(self, command, **kwargs):
 
     def get_packager(self):
         if not self.packager:
-            self.packager = self.run(PACKAGER_COMMAND, verbose=False).stdout.strip()
+            self.packager = self.run(conf["generic"]["packager_cmd"], verbose=False).stdout.strip()
```

Comment:
It will be specified always, right? What if packager_cmd is missing?

Reply:
it causes traceback. by default it is defined in shipped mtf config. if user remove this. then it will lead to expected traceback.